server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -226,14 +227,17 @@ static AccumuloConfiguration createCompactionConfiguration(Tablet tablet,
         cfg.getClassName(), CompactionConfigurer.class);
 
     configurer.init(new CompactionConfigurer.InitParamaters() {
+
+      private final ServiceEnvironment senv = new ServiceEnvironmentImpl(tablet.getContext());
+
```

Comment:
This is created twice in the createCompactionConfiguration method, once for the anonymous InitParamaters (which I just noticed is misspelled; should be InitParameters) and once for the anonymous InputParameters below. This could be created once in the method as a final local variable, and referenced in each anonymous inner class instance's getEnvironment method.

Reply:
> which I just noticed is misspelled; should be InitParameters

nice catch, that is an unreleased new API